cmd/krel/cmd/ff.go

```diff
@@ -115,58 +115,53 @@ func runFf(opts *ffOptions) error {
 		return err
 	}
 
+	if err := repo.CheckoutBranch(branch); err != nil {
+		return err
+	}
+
 	baseDir, err := ioutil.TempDir("", "ff-")
 	if err != nil {
 		return err
 	}
 
 	cleanup := rootOpts.cleanup
 	if cleanup {
-		defer cleanupTmpDir(baseDir)
+		defer repo.Cleanup()         //nolint: errcheck
+		defer cleanupTmpDir(baseDir) //nolint: errcheck
 	}
 
 	workingDir := filepath.Join(baseDir, branch)
-	log.Printf("%s", workingDir)
+	log.Printf("working directory is %q", workingDir)
 
 	os.Setenv("GOPATH", workingDir)
 	log.Printf("GOPATH: %s", os.Getenv("GOPATH"))
 
-	gitRoot := fmt.Sprintf("%s/src/k8s.io/kubernetes", workingDir)
-
-	// TODO: nomock?
-	if nomock {
-		log.Printf("nomock mode (from within ff)\n")
+	gitRoot := fmt.Sprintf("%s/src/k8s.io", workingDir)
+	if err := os.MkdirAll(gitRoot, 0o755); err != nil {
```

Comment:
Is this (`0o775`) a typo?

Reply:
This is new in go 1.13, see https://golang.org/ref/spec#Integer_literals

I assume this is fine since we're only supporting this version, right? :innocent:

Reply:
Ohhhh, that's cool! TIL :)
...and yes, onwards to go1.13!